docker-images/cts-base/README

```diff
@@ -1,5 +1,35 @@
 
+This directory contains the Dockerfiles used for creating images used to run the TCK tests in the Eclipse CI using Jenkins pipeline.
+-Use eclipse bugzilla (https://bugs.eclipse.org/bugs) to avail write permissions on any repositories in these docker hub organizations
+-If the images are required to be updated, create new tags(versions) of the same docker image ; it is possible to overwrite the existing image:version but not recommended.
+-If the docker image is updated with another version the same has to be updated in Jenkinsfile to use them in the CI runs.
+
+=====================================
+
+Dockerfile1:
+
+Image - jakartaee/cts-base:0.1
+
 Additional softwares to be downloaded
-=======================================
-j2sdk-1_4_2_41-linux-i586.tar.gz
-jdk-8u191-linux-x64.tar.gz
+-j2sdk-1_4_2_41-linux-i586.tar.gz
+-jdk-8u191-linux-x64.tar.gz
+
+Build command : docker build -t jakartaee/cts-base:0.1 -f Dockerfile1 .
+Image push command : docker push jakartaee/cts-base:0.1
+
+=====================================
+
+Dockerfile2:
+
+Image - jakartaee/cts-base:0.2
+
+Additional softwares to be downloaded
+-jdk-11.0.7_linux-x64_bin.tar.gz
+-jdk-8u191-linux-x64.tar.gz
+
+JDK11_HOME will the available as JDK11 home path at /opt/jdk-11.0.7
```

Comment:
If/when we add JDK updates in the future, say jdk-11.0.8_linux-x64_bin.tar.gz, would we have both /opt/jdk-11.0.7 + /opt/jdk-11.0.8 available?

Reply:
When jdk-11.0.8_linux becomes available and if it is required to be used for our TCK runs, then one of us will have to etiher
1. create a new version of the jakartaee/cts-base image(with tag 0.3) having jdk-11.0.8_linux (& jdk11.0.7 removed)
OR 
2. update the jakartaee/cts-base:0.2 with jdk-11.0.8_linux in place of jdk-11.0.7_linux 

IMHO to answer when to create a new tag(version) of the docker image :  
- if we have substantial changes we can create a new(version/tag) docker image. 
-Also if we have a stable running present version, we shouldn't  create new one for future changes.
Otherwise I think it should be ok to keep updating the same tag/version of the image.


Dockerfile1 : corresponds to jakartaee/cts-base:0.1 and has jdk8_191; been used for all our jakartaee-tck CI runs until now.
Dockerfile2 :  corresponds to jakartaee/cts-base:0.2( created today) has jdk11.0.7 and other changes as in this PR
For future versions or changes , eg:
Dockerfile3: can correspond to jakartaee/cts-base:0.3 (with future changes like jdk update, etc)

Reply:
@scottmarlow Please let me know if I need to add any more details to the README. I was thinking to have only the README for updating docker images.